### ISSUE TITLE
MAE of bigdl.optim.optimizer and zoo.pipeline.api.keras.objectives  are confusing

### ISSUE THREAD

**Issue by songhappy** [intel-analytics/analytics-zoo#1074]
MAE of bigdl.optim.optimizer is a validation method, and in zoo.pipeline.api.keras.objectives, mae = MAE = MeanAbsoluteError, then MAE is a criterion.
There are a lot of places we do  
```
from zoo.pipeline.api.keras.objectives import *
from bigdl.optim.optimizer import *
```
then MAE always calls the later one which cause problems.

**Comment by songhappy** [comment]
fixed by 
https://github.com/intel-analytics/analytics-zoo/pull/1075

**Comment by jason-dai** [comment]
Is MAE a metrics or loss in Keras? @zhichao-li

**Comment by zhichao-li** [comment]
`MAE` is not a metrics in Keras 1.2.2, but user can customize a similar one.
In Keras 2.x, user can use any loss function as metrics, maybe we can support that later. Theoretically, the forwarding logic is the same only that the interface is not the same in Zoo for now.
